### PR TITLE
Fix http transmitter exception

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

File: superdesk/publish/transmitters/http_push.py [hunk 1]
```diff
@@ -61,10 +61,18 @@ def _copy_published_media_files(self, item, assets_url):
             del renditions[name]['href']
             if not self._media_exists(rendition['media'], assets_url):
                 media = app.media.get(rendition['media'], resource='upload')
-                files = {'media': (rendition['media'], media, rendition.get('mimetype') or rendition['mime_type'])}
-                response = requests.post(assets_url, files=files, data={'media_id': rendition['media']})
+                files = {'media': (
+                    rendition['media'], media,
+                    rendition.get('mimetype') or rendition['mime_type']
```
Comment (mugurrus) [inline superdesk/publish/transmitters/http_push.py]: AFAIK only mimetype is used in renditions.

**Comment by mugurrus** [comment]
:+1:

File: superdesk/publish/transmitters/http_push.py (hunk 1 above)
Comment (actionless) [inline superdesk/publish/transmitters/http_push.py]: should i update it here then?
that commit was changing only the style

Comment (mugurrus) [inline superdesk/publish/transmitters/http_push.py]: I would update it in this commit as it's a very small change

Comment (actionless) [inline superdesk/publish/transmitters/http_push.py]: should i change it to just `rendition['mimetype']`?

Comment (mugurrus) [inline superdesk/publish/transmitters/http_push.py]: yep

**Pull request merged**